CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+Thank you for your interest in contributing to the Rust In Pieces!
+
+There are a few ways of helping: critiquing the documentation and/or code examples, fixing
+incorrect information, and fixing bugs.
+
+It is encouraged for you to run the tests, formatters, and linters before submitting PRs. They will
```

Comment:
I agree with you, adding the Rust/Python formatting guidelines are necessary because a lot of people will never have worked `pre-commit.yml` and the likes before (and they may not even have it installed).

Reply:
I added a section on using pre-commit. Let me know if these instructions explain it well for a new user to pre-commit.